From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Jelle Raaijmakers <jelle@gmta.nl>
Date: Sun, 11 Feb 2024 22:25:59 +0100
Subject: [PATCH] Build: Remove pkgconfig

We can't use pkgconfig here since the OpenGL dependency we find is
incompatible with it as an argument.
---
 meson.build | 8 --------
 1 file changed, 8 deletions(-)

diff --git a/meson.build b/meson.build
index 7949e300ac60e0ff6955dedfd9a050cfa3010f4a..eed52aea0419aa4efa7ae6d4e71cac4bc28a7b79 100644
--- a/meson.build
+++ b/meson.build
@@ -28,11 +28,3 @@ install_headers(
   'include/GL/glu.h',
   subdir : 'GL',
 )
-
-pkgconf = import('pkgconfig')
-pkgconf.generate(
-  libglu,
-  name : 'glu',
-  description : 'Mesa OpenGL Utility Library',
-  requires: dep_gl
-)
